ESSArch_Core/frontend/styles/core/modules/modals.scss

```diff
@@ -0,0 +1,23 @@
+// Remove scroll on main page when modal is open
+body.modal-open {
+    overflow: hidden;
+}
+
+body.modal-open[style] {
+    padding-right: 0px !important;
+}
+
+.modal::-webkit-scrollbar {
```

Comment:
This is not a standard and will not work in all browsers. IMO we don't need it at all.